src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import './TextArea.css'
+
+    type NameFormProps={
+        placeholder?:string,
+        className ?:string
+        disabled?: boolean
+    
+    }
+   export const TextAreaForm = (props:NameFormProps) => {
```

Comment:
Название компонента должно соответствовать названию файла
```suggestion
   export const TextArea = (props:NameFormProps) => {
```